src/test/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/connection/TestConnectionManager.kt

```diff
@@ -893,79 +873,6 @@ class TestConnectionManager {
             }
     }
 
-    @Test
-    fun `connection manager handles ack timeout and subscription cancel`() {
```

Comment:
@lumber1000 could you please advise why have you removed this test?

Reply:
this test is included into `connection manager handles ack timeout (multiple subscribers in parallel)` as second test case ("queue2").